apps/mux/src/config.tsx

```diff
@@ -0,0 +1,292 @@
+/* eslint-disable jsx-a11y/media-has-caption */
+
+import React from 'react';
+
+import '@contentful/forma-36-react-components/dist/styles.css';
+import '@contentful/forma-36-fcss/dist/styles.css';
+import {
+  Heading,
+  Subheading,
+  Note,
+  Form,
+  TextField,
+  CheckboxField,
+  Paragraph,
+  Typography,
+  FieldGroup,
+  TextLink,
+} from '@contentful/forma-36-react-components';
+import {
+  AppExtensionSDK,
+  AppConfigAPI,
+  SpaceAPI,
+  BaseExtensionSDK,
+} from 'contentful-ui-extensions-sdk';
```

Comment:
```suggestion
} from 'contentful-ui-extensions-sdk/lib/types';
```

Reply:
I committed this change, and it looks like this is causing CI to fail

```
FAIL test/index.test.tsx
  ● Test suite failed to run

    src/config.tsx:24:8 - error TS2307: Cannot find module 'contentful-ui-extensions-sdk/lib/types'.

    24 } from 'contentful-ui-extensions-sdk/lib/types';
              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```